raiden_contracts/deploy/__main__.py

```diff
@@ -664,55 +664,40 @@ def verify_deployed_contracts(web3: Web3, contract_manager: ContractManager, dep
     assert contract_manager.contracts_version == deployment_data['contracts_version']
     assert chain_id == deployment_data['chain_id']
 
-    endpoint_registry = verify_deployed_contract(
+    verify_deployed_contract(
         web3,
         contract_manager,
         deployment_data,
         CONTRACT_ENDPOINT_REGISTRY,
     )
-    print(
-        f'{CONTRACT_ENDPOINT_REGISTRY} at {endpoint_registry.address} '
-        f'matches the compiled data from contracts.json',
-    )
 
     secret_registry = verify_deployed_contract(
         web3,
         contract_manager,
         deployment_data,
         CONTRACT_SECRET_REGISTRY,
     )
-    print(
-        f'{CONTRACT_SECRET_REGISTRY} at {secret_registry.address} '
-        f'matches the compiled data from contracts.json',
-    )
 
-    token_network_registry = verify_deployed_contract(
+    def token_network_registry_checker(deploy_record, onchain_instance):
+        constructor_arguments = deploy_record['constructor_arguments']
+        assert to_checksum_address(
+            onchain_instance.functions.secret_registry_address().call(),
+        ) == secret_registry.address
+        assert secret_registry.address == constructor_arguments[0]
+
+        chain_id = onchain_instance.functions.chain_id().call()
+        assert chain_id == constructor_arguments[1]
+
+        settlement_timeout_min = onchain_instance.functions.settlement_timeout_min().call()
+        settlement_timeout_max = onchain_instance.functions.settlement_timeout_max().call()
+        assert settlement_timeout_min == constructor_arguments[2]
+        assert settlement_timeout_max == constructor_arguments[3]
```

Comment:
I would do the assert directly if it fits in a single line, but that's a matter of taste:
```python
assert onchain_instance.functions.settlement_timeout_max().call() == constructor_arguments[3]
```